sdk/spring/azure-spring-integration-servicebus/src/main/java/com/azure/spring/integration/servicebus/converter/ServiceBusMessageHeaders.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.integration.servicebus.converter;
+
+import com.azure.spring.integration.core.AzureHeaders;
+
+/**
+ * Azure service bus internal headers for Spring Messaging messages.
+ */
+public class ServiceBusMessageHeaders extends AzureHeaders {
+
+    private static final String PREFIX = AzureHeaders.PREFIX + "service_bus_";
+    /**
+     * There are 16 items can be set for service bus IMessage.
+     * (1). 2 items are deprecated: ScheduledEnqueuedTimeUtc, Body.
+     * (2). 3 items should be set by Spring message: ContentType, MessageBody. ReplyTo
+     * (3). 1 item should not be set: Properties.
+     * (4). The rest 10 items can be set by Spring message header:
```

Comment:
nice doc, probably add a link to the original service bus doc? so we can keep the reference and keep the code updated

Reply:
Good suggestion 👍 
Updated.